### PR TITLE
22.2: Ubuntu/impish

### PR DESCRIPTION
**Pull request by TheRealFalcon**

### CODE REVIEW

**Review by blackboxsw — CHANGES_REQUESTED** [review]
lintian spelling warning we should probably fix while we are here.

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,199 @@
...
+ - docs: Add docs for module creation (#1415)
+ - Remove cheetah from templater (#1416)
+ - tests: verify_ordered_items fallback to re.escape if needed (#1420)
+ - Misc module cleanup (#1418)
+ - docs: Fix doc warnings and enable errors (#1419)
+   [Alberto Contreras] (GH: #1876341)
+ - Refactor cloudinit.sources.NetworkConfigSource to enum (#1413)
+   [Alberto Contreras] (GH: #1874875)
+ - Don't fail if IB and Ethernet devices 'collide' (#1411)
+ - Use cc_* module meta defintion over hardcoded vars (SC-888) (#1385)
```
[the remark this answers is not in the document]
Comment (blackboxsw) [reply ChangeLog:50]: Warnings seen. The rest we probably should address sometime in the future, but not this PR.
```
csmith@downtown:/tmp/cloud-init (ubuntu/impish)$ lintian cloud-init_22.2-0ubuntu1~22.10.1_all.deb
W: cloud-init: command-with-path-in-maintainer-script /usr/sbin/grub-install (in test syntax) [postinst:312]
W: cloud-init: mismatched-override systemd-service-file-refers-to-unusual-wantedby-target lib/systemd/system/cloud-config.service cloud-init.target
W: cloud-init: mismatched-override systemd-service-file-refers-to-unusual-wantedby-target lib/systemd/system/cloud-final.service cloud-init.target
W: cloud-init: mismatched-override systemd-service-file-refers-to-unusual-wantedby-target lib/systemd/system/cloud-init-local.service cloud-init.target
W: cloud-init: mismatched-override ... use --no-tag-display-limit to see all (or pipe to a file/program)
W: cloud-init: spelling-error-in-changelog defintion definition
W: cloud-init: systemd-service-file-refers-to-unusual-wantedby-target cloud-init.target [lib/systemd/system/cloud-config.service]
W: cloud-init: systemd-service-file-refers-to-unusual-wantedby-target cloud-init.target [lib/systemd/system/cloud-final.service]
W: cloud-init: systemd-service-file-refers-to-unusual-wantedby-target cloud-init.target [lib/systemd/system/cloud-init-local.service]
W: cloud-init: systemd-service-file-refers-to-unusual-wantedby-target ... use --no-tag-display-limit to see all (or pipe to a file/program)
N: 5 hints overridden (5 info); 4 unused overrides
```

[the remark this answers is not in the document]
Comment (TheRealFalcon) [reply ChangeLog:50]: Heh, I actually noticed this when looking over the changelog, but figured we didn't want to modify something that came right of the git commits. Should be fixed now.

**Review by blackboxsw — APPROVED** [review]
Thanks for SRU bug fix in changelogs and the typo fix. +1 sbuild/lintian are happy.

I'll dput and push to upstream once dput succeeds.

**Pull request merged**